cellprofiler/modules/measureradialentropy.py

```diff
@@ -0,0 +1,205 @@
+'''<b>MeasureRadialEntropy</b> measures the variability of an image's
+intensity inside a certain object
+<hr>
+<p>MeasureRadialEntropy divides an object into pie-shaped wedges and
+ measures either the mean or median intensity of each.  Once the intensity
+ of each wedge has been calculated, the entropy of the bin totals is calculated.</p>
+
+ <p>This module is under construction</p>
+
+'''
+
+
+import numpy as np
+from scipy import stats
```

Comment:
Use `import scipy.stats` rather than `import scipy`. It can be confusing when multiple packages have identical module names (e.g. Python has a built-in module that has a similar name, `[statistics](https://docs.python.org/3/library/statistics.html)`).